### PR TITLE
Feature/dr 10550 file state feedback

### PR DESCRIPTION
**Pull request by aholzner**

### CODE REVIEW

**Comment by jayhof** [comment]
I've added @I2olanD as reviewer, as some of the changes were implemented by me.

**Comment by jayhof** [comment]
@aholzner Test coverage reports line 70 and 98 of VueBucketLoader.vue are uncovered, can you take a look at that?

**Comment by I2olanD** [comment]
@aholzner @jayhof Would it be a good idea to use our own joblocal style for the docs instead of the default bootstrap style?

**Comment by jayhof** [comment]
@I2olanD I specifically chose the default bootstrap to reach a broader audience - make it clear that style is not part of this package.

File: docs/src/components/App.vue [hunk 1]
```diff
@@ -86,50 +88,41 @@ export default {
   },
 
   data: () => ({
-    staticEndpointExample: `
-      <VueBucketLoader
-        :signingUrl="'https://httpbin.org/post'"
-      />`,
+    staticEndpointExample: `<VueBucketLoader
+  :signingUrl="'https://httpbin.org/post'"
+/>`,
```
Comment (I2olanD) [inline docs/src/components/App.vue]: @aholzner please check the format

**Comment by I2olanD** [comment]
@aholzner @jayhof  the script command for the package release has to be changed to just "release": "npm publish". Maybe consider to add the "npm version patch" command to it.

**Comment by jayhof** [comment]
@I2olanD can you explain why you'd remove --access public? If you check npm https://docs.npmjs.com/cli/publish they default to --access restricted. This is a public package though.

**Comment by I2olanD** [comment]
@jayhof i think this is only necessary for the initial release (package not registered to npm). Also it did not work for us in the last release and i published it via npm publish.

**Comment by I2olanD** [comment]
@aholzner @jayhof also there are conflicts in 'docs/src/components/App.vue'

**Comment by I2olanD** [comment]
@aholzner @jayhof could you also update the readme for the new upload status usage?

**Pull request merged**